core/arch/arm/plat-stm32mp1/plat_tzc400.c

```diff
@@ -84,6 +77,13 @@ static TEE_Result init_stm32mp1_tzc(void)
 	 * OP-TEE hence here only check this complies with static Core
 	 * expectations.
 	 */
+#define _DRAM_BEGIN		DDR_BASE
```

Comment:
Why are you not using local variables instead?

Reply:
fair :)